src/analysis/complex/isometry.lean

```diff
@@ -145,3 +146,24 @@ begin
   { simpa using eq_mul_of_inv_mul_eq h₁ },
   { exact eq_mul_of_inv_mul_eq h₂ }
 end
+
+/-- The matrix representation of `rotation`. -/
+lemma to_matrix_rotation (a : circle) :
+  linear_map.to_matrix basis_one_I basis_one_I (rotation a).to_linear_equiv =
+    ![![re a, -im a], ![im a, re a]] :=
```

Comment:
I think it might be worth using
```suggestion
    plane_conformal_matrix (re a) (-im a) sorry :=
```
here. @hrmacbeth, is the minus sign here expected, or have we defined `plane_conformal_matrix` as the transpose of the obvious choice?

Reply:
I've changed it to use `plane_conformal_matrix` (that does introduce a further dependency, on `linear_algebra.general_linear_group`, which wasn't previously imported here).

Reply:
If you don't mind, can you fix the sign in `plane_conformal_matrix` while you're at it?

Reply:
I've made that change (and consequent fix to `number_theory.modular`, the only place previously using that definition).